<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="">
      <property name="title">
       <string>Main windows</string>
      </property>
      <layout class="QHBoxLayout" name="Mainhorizontal">
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item>
        <widget class="QGroupBox" name="SetingsBox">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QVBoxLayout" name="ButtonsVerrtical">
          <item>
           <widget class="QGroupBox" name="RadioBox">
            <property name="title">
             <string>Encodings</string>
            </property>
            <layout class="QVBoxLayout" name="RadioVertical">
             <item>
              <widget class="QRadioButton" name="RadioCP1251">
               <property name="text">
                <string>cp1251</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioUTF8">
               <property name="text">
                <string>UTF-8</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="">
            <property name="title">
             <string>Text settings</string>
            </property>
            <layout class="QVBoxLayout" name="CheckBoxVertical">
             <item>
              <widget class="QCheckBox" name="checkBoxBoldFront">
               <property name="text">
                <string>Полужирный</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoItalics">
               <property name="text">
                <string>Курсив</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxUnderlined">
               <property name="text">
                <string>Подчеркнутый</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Черный</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Белый</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Красный</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Синий</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="ButtonBox">
      <property name="title">
       <string>Action buttons</string>
      </property>
      <layout class="QHBoxLayout" name="HorizontalButtons">
       <item>
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
